runtime/integration-tests/src/liquidity_pools/pallet/development/tests/liquidity_pools/foreign_investments.rs

```diff
@@ -3045,188 +3182,26 @@ mod mismatching_currencies {
-						currency_in: foreign_currency,
-						currency_out: pool_currency
-					}
-				}
-			);
-			// ExecutedCollectRedeem should not have been dispatched
-			assert!(System::events().iter().any(|e| {
-				match &e.event {
-					development_runtime::RuntimeEvent::LiquidityPoolsGateway(
-						pallet_liquidity_pools_gateway::Event::OutboundMessageSubmitted {
-							message,
-							..
-						},
-					) => match message {
-						pallet_liquidity_pools::Message::ExecutedCollectRedeem { .. } => false,
-						_ => true,
-					},
-					_ => true,
-				}
-			}));
-
-			// Process remaining redemption at 25% rate, i.e. 1 pool currency = 4 tranche
-			// tokens
-			assert_ok!(Investments::process_redeem_orders(default_investment_id()));
-			assert_ok!(Investments::redeem_fulfillment(
-				default_investment_id(),
-				FulfillmentWithPrice {
-					of_amount: Perquintill::from_percent(100),
-					price: Ratio::checked_from_rational(1, 4).unwrap(),
-				}
-			));
-			assert!(Investments::redemption_requires_collect(
-				&investor,
-				default_investment_id()
-			));
-			assert_ok!(OrderBook::fill_order_full(
-				RuntimeOrigin::signed(trader.clone()),
-				swap_order_id
-			));
-			assert!(!Investments::redemption_requires_collect(
-				&investor,
-				default_investment_id()
-			));
-			assert_eq!(
-				RedemptionState::<DevelopmentRuntime>::get(&investor, default_investment_id()),
-				RedeemState::ActiveSwapIntoForeignCurrencyAndSwapIntoForeignDone {
-					done_amount: redeem_amount_foreign_denominated / 4,
-					swap: Swap {
-						amount: redeem_amount_foreign_denominated / 8,
-						currency_in: foreign_currency,
-						currency_out: pool_currency
-					}
-				}
-			);
-			// ExecutedCollectRedeem should not have been dispatched as RedemptionState is
-			// still swapping
-			ensure_executed_collect_redeem_not_dispatched();
-
-			// Fulfill redemption swap
-			assert_ok!(OrderBook::fill_order_full(
-				RuntimeOrigin::signed(trader.clone()),
-				swap_order_id + 1
-			));
-			assert!(!RedemptionState::<DevelopmentRuntime>::contains_key(
-				&investor,
-				default_investment_id()
-			));
-			assert!(System::events().iter().any(|e| {
-				e.event
-					== pallet_liquidity_pools_gateway::Event::OutboundMessageSubmitted {
-						sender: TreasuryAccount::get(),
-						domain: DEFAULT_DOMAIN_ADDRESS_MOONBEAM.domain(),
-						message: pallet_liquidity_pools::Message::ExecutedCollectRedeem {
-							pool_id,
-							tranche_id: default_tranche_id(pool_id),
-							investor: investor.clone().into(),
-							currency: general_currency_index(foreign_currency),
-							currency_payout: redeem_amount_foreign_denominated / 8 * 3,
-							tranche_tokens_payout: redeem_amount_pool_denominated,
-							remaining_redeem_amount: 0,
-						},
-					}
-					.into()
-			}));
-		});
-	}
-
 	/// Similar to [concurrent_swap_orders_same_direction] but with partial
 	/// fulfillment
-	#[test]
-	fn partial_fulfillment_concurrent_swap_orders_same_direction() {
```

Comment:
Is this test gone?

Reply:
I'll have to double check this since I think William did some changes to these earlier.

Reply:
Nvm, I screwed these up, sorry about that. Fixed them, now I'm testing them locally.

Reply:
Fixed.